.golangci.yml

```diff
@@ -147,25 +136,20 @@ linters:
     - ineffassign
     - misspell
     - nakedret
-    - exportloopref
+    - copyloopvar
     - staticcheck
-    - structcheck
     - stylecheck
     - typecheck
     - unconvert
     - unparam
-    - varcheck
     - whitespace
 issues:
   exclude-rules:
-    - path: 'main.go'
-      linters:
-        - revive
-      text: 'package-comments: should have a package comment'
     - path: main.go
       linters:
         - funlen
-      text: "Function 'main'"
+        - gocritic
+        - revive
```

Comment:
Is it possible to disable just certain issues and keep these linters working with main.go?

Reply:
Sure, investigated more, we had collision of gocritic and govet linter about shadowing 'err' variables and as i got -  this is convention to re-assign 'err', so applied gocritic fix and excluded govet 'err' shadowing